showcase.json

```diff
@@ -3711,5 +3711,27 @@
     "stats": "1000+ Monthly Users",
     "display": "Normal",
     "logo": "/img/showcase/vaultbet_logo.webp"
+  },
+  {
+    "id": "icto",
+    "name": "ICTO",
+    "oneLiner": "ICTO is a one-stop platform automating token operations on the Internet Computer.",
+    "tags": [
+      "Tools / Infrastructure",
+      "NFT",
+      "DAO",
+      "DeFi"
+    ],
+    "description": "ICTO is an automated token management platform that simplifies vesting schedules, recurring payroll distributions, token lock and tokenized crowdfunding for projects building on the Internet Computer. It provides modular smart contracts, allowing any project to transparently handle token operations without technical friction.",
+    "usesInternetIdentity": true,
+    "website": "https://icto.app",
+    "github": "https://github.com/ICTO-Labs",
```

Comment:
There are no public repos in this org.

Reply:
Hi @domwoe 
Currently, we are refactoring the repos to bring them to the official organization: ICTO Labs. There was a requirement to have a public repo, so we published a repo related to the previous activity related to staking NFTs and exchanging for Tokens: https://github.com/ICTO-Labs/staking-pool
The current ICTO project, as you know, is in the process of being developed so it is temporarily not open. We commit to open the entire source code after completion (we have granted you viewing rights).